modules/npmscripts.config.js

```diff
@@ -155,6 +155,9 @@ module.exports = {
 					'@liferay/template-web': {
 						'/': '*',
 					},
+					'analytics-reports-js-components-web': {
```

Comment:
I think this also needs to be added to build.imports in this config file
![Screenshot 2024-08-14 at 1 53 52 PM](https://github.com/user-attachments/assets/0e3d45a4-4085-4fb1-8571-bfc25127d5c3)

Reply:
Done